### PR TITLE
Writer pass on "Support multiple profiles under a single user data folder"

### PR DESCRIPTION
**Pull request by mikehoffms**
This PR merges into PR https://github.com/MicrosoftDocs/edge-developer/pull/2245.
This PR makes changes:
* Writer/Editor/tagging pass.
* Added API Ref links.
* Uses latest tab-set tagging per Overview of APIs article.
* Uses h2 for major sections, h4 for minor headings not in Nav UI.

To review this PR:
* Click the tabs in https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/multi-profile-support?branch=pr-en-us-2260 and make sure they work.

### CODE REVIEW

**Review by leonhsl — APPROVED** [review]
Thanks Michael!

**Review by JasonAndrewWriter — APPROVED** [review]
There are a couple of ToDos in the code boxes.  Michael said they were being put in later. Otherwise looks good.

File: microsoft-edge/webview2/concepts/multi-profile-support.md [hunk 1]
```diff
@@ -6,84 +6,152 @@ ms.author: msedgedevrel
...
 
-# [C++](#tab/cpp)
 
-* **[ICoreWebView2Environment10](/microsoft-edge/webview2/reference/win32/icorewebview2environment10)**
-   * [CreateCoreWebView2ControllerWithOptions](/microsoft-edge/webview2/reference/win32/icorewebview2environment10#createcorewebview2controllerwithoptions)
-   * [CreateCoreWebView2CompositionControllerWithOptions](/microsoft-edge/webview2/reference/win32/icorewebview2environment10#createcorewebview2compositioncontrollerwithoptions)
+##### [.NET/C#](#tab/dotnetcsharp)
 
----
+<!-- TODO: replace code listing? -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/multi-profile-support.md:82]: @leonhsl - Replace the below C# code that's copied from spec?
WinRT section has same code.

File: microsoft-edge/webview2/concepts/multi-profile-support.md [hunk 2]
```diff
@@ -149,72 +217,109 @@ HRESULT AppWindow::CreateControllerWithOptions()
...
+<!-- ====================================================================== -->
+## Example of accessing and manipulating the profile
 
-[ICoreWebView2Profile Interface](/microsoft-edge/webview2/reference/win32/icorewebview2profile)
 
-[ICoreWebView2Profile2 Interface](/microsoft-edge/webview2/reference/win32/icorewebview2profile2)
+##### [.NET/C#](#tab/dotnetcsharp)
 
----
+<!-- todo: replace code listing? -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/multi-profile-support.md:264]: 2. @leonhsl - Replace the below C# code that's copied from spec?
WinRT section has same code.

**Pull request merged**